pages/about.html

```diff
@@ -0,0 +1,241 @@
+    <meta name="author" content="" />
+    <meta name="description" content="" />
+
+    <!-- Links -->
+    <link href="../css/about.css" rel="stylesheet" />
+    <link href="../css/sanitize.css" rel="stylesheet" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Tittle -->
+    <title>Capstone Project</title>
+
+    <!-- Font Awesome Icons -->
+    <script
+      src="https://kit.fontawesome.com/2f238780d4.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <!-- Script -->
+    <script src="../index.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <!-- Navigation mobile -->
+
+      <nav class="nav__mobile-menu">
+        <button class="nav__btn-mobile" type="button">
+          <i class="nav__icon-close fas fa-times"></i>
+        </button>
+        <ul class="nav__mobile">
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="../index.html">Home</a>
+          </li>
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="../pages/about.html">About</a>
+          </li>
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="#">Program</a>
+          </li>
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="#">Join</a>
+          </li>
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="#">Sponsor</a>
+          </li>
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="#">News</a>
+          </li>
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="#">Campaign</a>
+          </li>
+        </ul>
+      </nav>
+
+      <!-- Navigation desktop top -->
+
+      <nav class="nav-top">
+        <ul class="nav-top__ul">
+          <li>
+            <a class="nav-top__a" href=""><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a class="nav-top__a" href=""><i class="fab fa-twitter"></i></a>
+          </li>
+          <li><a class="nav-top__a" href="">English</a></li>
+          <li><a class="nav-top__a" href="">My Page</a></li>
+          <li><a class="nav-top__a" href="">Logout</a></li>
+        </ul>
+      </nav>
+
+      <!-- Navigation desktop bottom -->
+
+      <nav class="nav">
+        <div class="nav__container">
+          <a href="../index.html"
+            ><img
+              class="nav__logo"
+              src="../img/fintech-logo-1.svg"
+              alt="FinTech logo"
+          /></a>
+          <ul class="nav__ul">
+            <li>
+              <a class="nav__a" href="pages/about.html">About</a>
+            </li>
```

Comment:
- [ ] Consider fixing this path too so that your web page will work as expected and not crash 😄

```suggestion
            <li>
              <a class="nav__a" href="about.html">About</a>
            </li>
```